backend/app/models/item.py

```
@@ -0,0 +1,47 @@
+
+    name: str = Field(index=True, min_length=1, max_length=100)
+    description: Optional[str] = Field(default=None, max_length=500)
+    price: Decimal = Field(gt=0, decimal_places=2)
+    quantity: int = Field(ge=0, default=0)
+    is_available: bool = Field(default=True)
+
+
+class Item(ItemBase, BaseModel, table=True):
+    """Item database model."""
+
+    pass
```

Ruff (0.12.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)